_rules/meta-viewport-b4f0c3.md

```diff
@@ -0,0 +1,223 @@
+---
+id: b4f0c3
+name: meta viewport does not prevent zoom
+rule_type: atomic
+description: |
+  This rule checks that the `meta` element is not used to block the user agent ability to zoom. 
+accessibility_requirements:
+  wcag21:1.4.4: # Resize text (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+authors:
+  - Jey Nandakumar
+  - Audrey Maniez
+---
+
+## Applicability
+
+The rule applies to each [`<meta>`](https://www.w3.org/TR/html52/document-metadata.html#the-meta-element) element with a [`name`](https://www.w3.org/TR/html52/document-metadata.html#element-attrdef-meta-name) attribute whose value is set to [`viewport`](https://developer.mozilla.org/en-US/docs/Mozilla/Mobile/Viewport_meta_tag#Enter_viewport_meta_tag) and has a [`content`](https://www.w3.org/TR/html52/document-metadata.html#dom-htmlmetaelement-content) attribute.
+
+## Expectation
+
+The [`content`](https://www.w3.org/TR/html52/document-metadata.html#dom-htmlmetaelement-content) attribute, whose value is mapped to a list of property/value pairs in a user-agent specific manner, does not:
+
+- specify the property `user-scalable` with a value of `no`; nor
+- specify the property `maximum-scale` with a value of less than 2
+
+**Note:** For Safari on the iPhone, the content attribute is mapped to property/value pairs according to the following algorithm: https://drafts.csswg.org/css-device-adapt/#parsing-algorithm
+
+## Assumptions
+
+- The [page](https://www.w3.org/TR/WCAG21/#dfn-web-page-s) has [visible](#visible) [content](https://www.w3.org/TR/WCAG21/#dfn-content).
+
+## Accessibility Support
+
+- Interpretation of `<meta name="viewport">` may vary with browsers, meaning browsers may override `user-scalable=no` and `maximum-scale=1` values to support gestures like pinch and zoom based interaction on permissible devices, thereby causing inconsistent behavior.
+
+## Background
+
+- [Understanding Success Criterion 1.4.4: Resize text](https://www.w3.org/WAI/WCAG21/Understanding/resize-text)
+- [The `meta` element](https://www.w3.org/TR/html52/document-metadata.html#the-meta-element)
+- [<meta>: The Document-level Metadata element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+The `<meta name="viewport">` element does not define the `maximum-scale` and `user-scalable` values.
```

Comment:
Not going to change this @Jym77. Let us try to have a unified consensus here first before making another change. Perhaps make an issue to discuss please.

Reply:
I am rather in favour of using `<element>` instead of `element`, but it seems that the huge majority of rules are doing it the other way, with only 4 using `<element>`:
```
 (develop=)$ grep -lr \`\< _rules/
_rules/html-title-is-descriptive-c4a8a4.md
_rules/link-has-name-c487ae.md
_rules/links-with-identical-names-serve-equivalent-purpose-b20e66.md
_rules/meta-refresh-bc659a.md
```

Reply:
I agree with JY on this one. This goes against convention. This is one we definitely did settle.